doc/api/events.md

```diff
@@ -180,6 +190,8 @@ As a best practice, developers should always register listeners for the
 `'error'` event:
 
 ```js
+'use strict';
+
```

Comment:
This example both works without `use strict` and is partial (a continuation of the first example), that means that this is not the correct place for `use strict`.